static/src/stylesheets/inline/article-labour-liverpool.scss

```diff
@@ -1,15 +1,27 @@
 @import '../_vars';
 @import '../_mixins';
-
 $labour-liverpool-header-element-opacity: .85;
 
+// make bottom left corner of headline flush with the bottom of the main media image
+$labour-liverpool-headline-top-supports-cover-mobile: 319px;
```

Comment:
Why make new variables? As far as I can tell, these are all only being used once? I think it would make more sense to just use the number and add a descriptive comment as to why